doc/python/probe_sample.rst

```diff
@@ -5,14 +5,13 @@ Cable cell probing and sampling
 
 .. module:: arbor
 
-Cable cell probe addresses are defined analogously to their counterparts in
+Cable cell probesets are defined analogously to their counterparts in
 the C++ API (see :ref:`cablecell-probes` for details). Sample data recorded
```

Comment:
This comment is very helpful :D

Reply:
It is a veiled excuse for the apparent convolution of the design. As per Gitter, the choices made stem from consideration, so I wanted to at least link to the outcomes of that consideration. I like that C++ and Python API are largely similar, but this is a good candidate for deviation. It is already, e.g. the `probe_tag` is nowhere to be found in Python. The `probe` class is not anything you actually can use to build with.

Should I do better than give excuses here? That would be to have people be able to create probes using only Python, and to maybe simplify things. Reading Gitter is sounds like that may be more involved than it seems, but maybe not? Maybe in a followup PR, if and when users ask for it?